java/app/src/main/java/com/fullstorydev/shoppedemo/ui/cart/CartFragment.java

```diff
@@ -21,13 +21,7 @@ public View onCreateView(@NonNull LayoutInflater inflater,
         cartViewModel =
```

Comment:
`cartViewModel` is instantiated but it doesn't appear to be used?

Reply:
added comment regarding this in marketViewModel #12